modules/runtime/pages/install-a-bonita-bpm-cluster.adoc

```diff
@@ -68,6 +63,12 @@ bonita.platform.cluster.hazelcast.tcpip.members=ipServer01,ipServer02
 +
 IMPORTANT: If you don't use `bonita.platform.cluster.hazelcast.multicast.enabled`, *you must uncomment the # properties and set it to `false`* to deactivate it, as follows: `bonita.platform.cluster.hazelcast.multicast.enabled=false`.
 +
+
+[NOTE]
+====
+The https://www.google.com/search?q=Hibernate+L2+cache[Hibernate L2 cache] will be automatically disabled in cluster, leading to lower performance and more frequent database accesses.
```

Comment:
Do we have details about that lower performance? isn't it a bit compensated in some way by adding more resources with new cluster nodes?

Reply:
I'll reword it to avoid making it sound dramatic